runtime/reconcilers/project_parser.go

```diff
@@ -300,7 +300,7 @@ func (r *ProjectParserReconciler) reconcileParser(ctx context.Context, inst *dri
 	}
 
 	// not setting restartController=true when diff is actually nil prevents infinite restarts
-	updateConfig := diff == nil || diff.ModifiedDotEnv || diff.Reloaded
+	updateConfig := diff == nil || diff.Reloaded
```

Comment:
Isn't it important to have the `diff.ModifiedDotEnv` here? Otherwise how will it update the variables when `.env` is interactively edited?